roles/postgresql_upgrade/tasks/main.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Ensure running on EL8
+  assert:
+    that:
+      - ansible_os_family == "RedHat"
+      - ansible_distribution_major_version == '8'
+    fail_msg: "This role is only supported on EL8"
+
+- name: Get version from the PostgreSQL data directory
+  slurp:
+    path: /var/lib/pgsql/data/PG_VERSION
+  register: pg_version
+  when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version == '8'
+
+- when:
+    - ansible_os_family == "RedHat"
+    - ansible_distribution_major_version == '8'
```

Comment:
```suggestion
```

The `assert` above will already have bailed out if that's not the case

Reply:
This doesn't hurt though :)